Makefile

```diff
@@ -20,10 +20,10 @@ swagger:
 	scripts/generate_swagger.sh
 
 swagger_client: swagger
-	cd ui && yarn && npm run build_api_client
+	cd ui && yarn install --update-checksums && npm run build_api_client
```

Comment:
Do we need `--frozen-lockfile` instead?

Reply:
Nope, adding `--update-checksums` is used to fix ["Integrity checked failed error"](https://github.com/yarnpkg/yarn/issues/7552) for yarn

Reply:
It seems `--frozen-lockfile` option works fine as well.